tests/rptest/tests/audit_log_test.py

```diff
@@ -511,6 +512,35 @@ def find_matching_record(self, filter_fn, valid_check_fn, desc):
         return records
 
 
+class AuditLogTestsAppLifecycle(AuditLogTestsBase):
+    """Validates that app lifecycle events occur
+    """
+    def __init__(self, test_context):
+        super(AuditLogTestsAppLifecycle,
+              self).__init__(test_context=test_context,
+                             audit_log_config=AuditLogConfig(event_types=[]))
+
+    @cluster(num_nodes=5)
+    def test_app_lifecycle(self):
+        def is_lifecycle_match(feature: Optional[str], is_start: bool, record):
+            expected_activity_id = 3 if is_start else 4
+
+            return record['class_uid'] == 6002 and record[
+                'activity_id'] == expected_activity_id and (
+                    (feature is not None and 'feature' in record['app']
+                     and record['app']['feature']['name'] == feature) or
+                    (feature is None and 'feature' not in record['app']))
+
+        _ = self.find_matching_record(
+            partial(is_lifecycle_match, "Audit System",
+                    True), lambda record_count: record_count == 3,
+            "Single redpanda audit start event per node")
+
+        _ = self.find_matching_record(partial(is_lifecycle_match, None, True),
```

Comment:
question: I wonder if you called `self.redpanda.stop_node()` with the clean=True parameter could you assert on seeing that event in the log or not

Reply:
I don't think we'l get a stop event until we have the audit system wait to stop the client after the sharded queues are drained